src/Common/src/System/Security/Cryptography/DSAOpenSsl.cs

```diff
@@ -197,9 +197,9 @@ public override byte[] CreateSignature(byte[] hash)
                 }
 
                 Debug.Assert(
-                    signatureSize == signature.Length,
+                    signatureSize <= signature.Length,
```

Comment:
Do we have the same problem on ECDSA and we're just getting lucky? Or was ECDSA already &lt;=?